ar_module2_7.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+var myDate = (function iffe() {
+  /* eslint-disable no-console */
+  var privateDate = new Date(); // make this available to both get and set
```

Comment:
Not a deal-breaker, but it would be preferable to set your privateDate variable to null to begin with. As soon as your module executes, it will set your date. In practice, your other methods should check it's null then set the date if it hasn't been yet, the net effect being that your internally stored date reflects when the module get's used, not loaded.